src/components/NestedEditComponent.tsx

```diff
@@ -311,13 +315,31 @@ export class NestedEditComponent extends React.Component<InlineComponentProps, a
     }
 
     select = (field: any, eventKey: any) => {
-        this.props.modelChanged(Object.assign({}, this.props.currentModel, { [field.field]: eventKey }))
+        if (this.props.index >= 0) {
```

Comment:
Added condition (this.props.index >=0). to support both iterable nested and nested. (without using .bind)